### ISSUE TITLE
restart: always for mysql & phpmyadmin

### ISSUE THREAD

**Issue by RobMaple** [litespeedtech/ols-docker-env#17]
Not sure if there is a reason for this but should the the mysql and phpmyadmin services have a restart policy added so that they restart on server reboot per the litespeed service?

**Comment by Code-Egg** [comment]
In my opinion, those two services are individual and no need a hook with LiteSpeed service. Or if there's any reason for that?

**Comment by RobMaple** [comment]
Sorry I think I phrased that badly - when I said 'server reboot' I meant the host machine rebooting.

Currently if the host machine restarts, the litespeed service/server restarts too due to the `restart:always` policy in the docker-compose file. Mysql and phpadmin don't however. 

This means that if the host machine restarts for whatever reason, any hosted sites are automatically accessible again once the litespeed container has resumed although the database is inaccessable until the mysql container is manually started. This obviously leads to default wordpress 'can't connect to database' status pages showing.

I've just added `restart:always` to both the mysql and phpmyadmin services to solve the issue although it seemed like it should be in the docker-compose file by defualt -- unless I'm missing something and this is not advisable for some reason?
